packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -670,6 +670,99 @@ Expected response body:
 }
 ```
 
+### Client_Naming_header
+
+- Endpoint: `post /projection/client-name-and-encoded-name/header`
+
+Testing that we can project a header name.
+Your generated SDK should generate an operation header `parameter` with a single parameter called `clientName`.
+
+Expected header parameter: `default-name="true"`
+Expected response header: `default-name="true"`
+
+### Client_Naming_Model_client
+
+- Endpoint: `post /projection/client-name-and-encoded-name/model/client`
+
+Testing that we can project the client name in our generated SDKs.
+Your generated SDK should generate the model with name `ClientModel`.
+
+Expected request body:
+
+```json
+{ "defaultName": true }
+```
+
+### Client_Naming_Model_language
+
+- Endpoint: `post /projection/client-name-and-encoded-name/model/language`
+
+Testing that we can project the language specific name in our generated SDKs.
+Your generated SDK should generate the model with your language specific model name.
+
+Expected request body:
+
+```json
+{ "defaultName": true }
+```
+
+### Client_Naming_operation
+
+- Endpoint: `post /projection/client-name-and-encoded-name/operation`
+
+Testing that we can project the operation name.
+Your generated SDK should generate an operation called `clientName`.
+
+Expected status code: 204
+
+### Client_Naming_parameter
+
+- Endpoint: `post /projection/client-name-and-encoded-name/parameter`
+
+Testing that we can project a parameter name.
+Your generated SDK should generate an operation `parameter` with a single parameter called `clientName`.
+
+Expected query parameter: `defaultName="true"`
+
+### Client_Naming_Property_client
+
+- Endpoint: `post /projection/client-name-and-encoded-name/property/client`
+
+Testing that we can project the client name in our generated SDKs.
+Your generated SDK should generate ClientNameModel with one property `clientName` with wire name `defaultName`.
+
+Expected request body:
+
+```json
+{ "defaultName": true }
+```
+
+### Client_Naming_Property_compatibleWithEncodedName
+
+- Endpoint: `post /projection/client-name-and-encoded-name/property/compatible-with-encoded-name`
+
+Testing that we can project the client name and the wire name.
+Your generated SDK should generate ClientNameAndJsonEncodedNameModel with one property with client name `clientName` and wire name `wireName`.
+
+Expected request body:
+
+```json
+{ "wireName": true }
+```
+
+### Client_Naming_Property_language
```

Comment:
the endpoints for these new tests still show the projection/client-name-and-encoded-name url. change the urls, and grep "project" and change it to rename so we completely remove "projection"